tests/61push/01message-pushed.pl

```diff
@@ -40,11 +40,21 @@
             })->SyTest::pass_on_done( "Bob received invite" ),
 
             matrix_invite_user_to_room( $alice, $bob, $room_id ),
+            flush_events_for( $alice ),
          )
       })->then( sub {
          # Bob accepts the invite by joining the room
          matrix_join_room( $bob, $room_id )
       })->then( sub {
+         await_event_for( $alice, filter => sub {
+            return 1;
```

Comment:
You probably don't want to wait for any possible event, because that leads to fragile tests that get upset about things like spurious `m.presence` updates. Consider looking for just `m.room.receipt` events maybe?